_search-plugins/star-tree-index.md

```diff
@@ -0,0 +1,189 @@
+- After the star-tree index is enabled for an index, it cannot be disabled. In order to disable the star-tree index, the data in the index must be re-indexed without the star-tree mapping. Furthermore, changing a star-tree configuration will also require a re-index operation.
+- [Multi-values/array values]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/index/#arrays) are not supported
+- Only [limited queries and aggregations](#supported-query-and-aggregations) are supported. Support for more features will be released in upcoming versions.
+- The cardinality of the dimensions should not be very high (like `_id` fields). High cardinality leads to higher storage usage and higher query latency.
+
+## Star-tree index structure
+
+The following image illustrates a standard star-tree index structure.
+
+<img src="{{site.url}}{{site.baseurl}}/images/star-tree-index.png" alt="A star-tree index containing two dimensions and two metrics" width="700">
+
+Sorted and aggregated star-tree documents are backed by `doc-values` in an index. Values follow the following patternL
+
+- The values are sorted based on the order of `ordered_dimension`, in the previous example, first by `status` and then by `port` for each of the status.
+- For each unique dimension value combination, the aggregated values for all the metrics such as `avg(size)` and `count(requests)` are pre-computed during ingestion time.
+
+### Leaf nodes
+
+Each node in the star-tree index points to a range of star-tree documents. Nodes can be further split into child nodes based on the[max_leaf_docs configuration]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/star-tree/#star-tree-configuration-parameters). The number of documents a leaf node points to is less than or equal to the number `max_leaf_docs` set. This ensures the maximum number of documents that traverse across nodes to get to the aggregated value is at most the `max_leaf_docs`, which provides predictable latency.
+
+### Star nodes
+
+`star nodes (*)` fetch aggregated document whenever applicable during query time.
+
+The star-tree index structure contains three examples explaining how documents traverse star-tree nodes during a `Term` query, based on the compute average request size
+
+Support for the `Term` query will be added in upcoming release. For more information, see [GitHub issue #15257](https://github.com/opensearch-project/OpenSearch/issues/15257).
+{: .note}
+
+- Where port the equals `8443` and status equals `200`. Because the status equals `200`, the document does not traverse through a star node.
+- Where status equals `200`. The query traverses through a star node in the `port` dimension, since `port` is not present as part of query.
+- Where port equals `5600`. The query traverses through star node in the `status` dimension, since `status` is not present as part of query. 
```

Comment:
```suggestion
- Where port equals `5600`. The query traverses through a star node in the `status` dimension, since `status` is not present as part of query. 
```